content/yaml-running-builds/starting-builds-automatically.md

```diff
@@ -321,20 +335,20 @@ workflows:
       condition: not event.pull_request.draft
 {{< /highlight >}}
 
-Example 2. This build will *not* continue if the pull request branch has a specific label name, "codemagicTest" in the following sample:
+**Example 2**. This build will *not* continue if the new label added to a pull request is not a specific label name, "codemagicTest" in the following sample:
```

Comment:
This explanation doesn't seem to go with the example as you have a double negative here ("will not continue" and "is not a specific label name").
```suggestion
**Example 2**. This build is triggered on adding a label to a pull request but will continue only if the label added was anything else than "codemagicTest":
```